mne/io/eeglab/eeglab.py

```diff
@@ -226,6 +233,12 @@ class RawEEGLAB(_BaseRaw):
         Names or indices of channels that should be designated
         EOG channels. If 'auto', the channel names containing
         ``EOG`` or ``EYE`` are used. Defaults to empty tuple.
+    event_id : dict
+        By default, events are read from the input file by dropping every
+        non-integer part of events containing integers, and completely
+        dropping any events without integer parts. If non-integer events
+        should be read, this should be a dict mapping from their names to
+        integers, e.g. dict(fmri_scan_onset=199, recording_start=255).
```

Comment:
@agramfort @jona-sassenhagen are we converging? @jona-sassenhagen whatever solution you have, can you make it work first? You have a failing test: https://ci.appveyor.com/project/Eric89GXL/mne-python/build/1.0.4561#L695 and it's not because of `preload`